dist/apb.js

```diff
@@ -82,13 +83,14 @@ var apb = /** @class */ (function () {
         });
     };
     //* BOOLEAN CHECKS & Validators /////////////////////////////////////////////////////
-    apb.prototype.isStateIntegration = function (stateName, States) {
+    apb.prototype.isInteractionStateWithParameters = function (stateName, States) {
         if (States === void 0) { States = this.States; }
         if (States[stateName] === undefined) {
             throw new Error("State " + stateName + " does not exist in the States object");
         }
         var state_to_check = States[stateName];
-        return ((state_to_check.Type === "Task" || state_to_check.Type === "Interaction") && !!state_to_check['Parameters']);
+        // return ((state_to_check.Type === "Task" || state_to_check.Type === "Interaction") && !!state_to_check['Parameters'])
+        return (state_to_check.Type === "Interaction" && !!state_to_check['Parameters']);
```

Comment:
question: This seems to have changed from a check for `Integration` to a check for `Interaction`. Is that intentional? Is the Integration check still happening somewhere else in the code?